src/sentry/api/endpoints/group_event_details.py

```diff
@@ -31,7 +32,7 @@
 
 
 def issue_search_query_to_conditions(
-    query: str, group: Group, user: User, environments: Sequence[Environment]
+    query: str, group: Group, user: User | AnonymousUser, environments: Sequence[Environment]
```

Comment:
in practice i don't think an AnonymousUser would ever make it in here, as to access the endpoint the user would have to be authed, although I don't think we have a way of defining the user type based on the type of request atm.

Reply:
ah: https://github.com/typeddjango/django-stubs#how-can-i-create-a-httprequest-thats-guaranteed-to-have-an-authenticated-user